pages.fr/common/adb.md

```diff
@@ -0,0 +1,33 @@
+# adb
+
+> Android Debug Bridge: Communiquer avec une instance d'émulateur Android ou un appareil Android.
+> Certaines commandes comme `adb shell` ont leur propre documentation.
```

Comment:
@Nico385412 would be great if you could add this phrase to the [translation template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#fr).

Reply:
done :D 
https://github.com/tldr-pages/tldr/pull/6820